### PR TITLE
chore: add issue template for WebIDL syntax changes

### PR DESCRIPTION
**Pull request by sidvishnoi**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/syntax_update.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: Web IDL Syntax Change
+about: Let us know if there are changes in WebIDL syntax
+title: ""
```
Comment (saschanaz) [inline .github/ISSUE_TEMPLATE/syntax_update.md:4]: Maybe this should start with BREAKING CHANGE:?

File: .github/ISSUE_TEMPLATE/syntax_update.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+assignees: ""
+--- 
+
+<!-- Required: URL to issue/PR in WebIDL repo -->
+URL: 
+
+<!-- Add any further details as needed -->
+<!-- Please prefix the issue title with "BREAKING CHANGE: " for breaking changes -->
+
+cc: @marcoscaceres @sidvishnoi for ReSpec
```
Comment (saschanaz) [inline .github/ISSUE_TEMPLATE/syntax_update.md:15]: Hehe 😁

**Review by saschanaz — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/syntax_update.md (hunk 1 above)
Comment (sidvishnoi) [reply .github/ISSUE_TEMPLATE/syntax_update.md:4]: I was thinking someone might skip the title if it is prefilled with something. WDYT?

File: .github/ISSUE_TEMPLATE/syntax_update.md (hunk 2 above)
Comment (sidvishnoi) [reply .github/ISSUE_TEMPLATE/syntax_update.md:15]: That's the goal :grin:

**Comment by sidvishnoi** [comment]
@saschanaz Can you merge this?

File: .github/ISSUE_TEMPLATE/syntax_update.md (hunk 1 above)
Comment (saschanaz) [reply .github/ISSUE_TEMPLATE/syntax_update.md:4]: I think a title ending with a colon explicitly requires adding more details. If anyone misses it, that means it's possibly just a spam.

File: .github/ISSUE_TEMPLATE/syntax_update.md [hunk 3]
```diff
@@ -10,6 +10,6 @@ assignees: ""
 URL: 
 
 <!-- Add any further details as needed -->
-<!-- Please prefix the issue title with "BREAKING CHANGE: " for breaking changes -->
+
```
Comment (saschanaz) [inline .github/ISSUE_TEMPLATE/syntax_update.md:13]:
```suggestion
```

**Review by saschanaz — APPROVED** [review]

**Comment by saschanaz** [comment]
👍

**Pull request merged**